docs/user/README.md

```diff
@@ -1,5 +1,5 @@
 > **TIP:** Apart from the {Module Name} heading, you can use your own titles for the remaining sections. You can also add more module-specific sections.
-
+<!--to be removed if the docs/README stays; should the whole user folder be deleted (for the time being)?-->
```

Comment:
I think user can be deleted for now and we can add it back once we can provide recommended usage details to the user.